bigtable/metrics.go

```diff
@@ -125,6 +125,7 @@ var (
 	// createExporterOptions takes Bigtable client options and returns exporter options
 	// Overwritten in tests
 	createExporterOptions = func(btOpts ...option.ClientOption) []option.ClientOption {
+		btOpts = append(btOpts, option.WithGRPCConnectionPool(4))
```

Comment:
do we need 4 channels for this? I think maximum 2 is fine cause there's only 1 RPC per minute.

Reply:
why do we need a channel pool for a client that will do 1 rpc per minute?

Reply:
Removed